ModularTegustation/fishing/code/fish/LC13_fish/saltwater_fish.dm

```diff
@@ -60,3 +60,20 @@
 		/datum/reagent/consumable/nutriment/organ_tissue = 1,
 		/datum/reagent/consumable/salt = 8,
 	)
+
```

Comment:
Going to request atomization for this, it's a small thing that can be merged almost instantly and doesn't really fit in this Gasharpoon PR.